### ISSUE TITLE
Address comment in PR #99

### ISSUE THREAD

**Issue by sappenin** [hyperledger/quilt#134]
See Adrian's comment in #99 [here](https://github.com/hyperledger/quilt/pull/99#discussion_r166477518)

**Comment by sappenin** [comment]
ilp-core/src/main/java/org/interledger/core/InterledgerProtocolException.java

```
public String toString(){
  return interledgerRejectPacket.toString();
}
```

As discussed in #89 I think this should include the value of this.getMessage()

**Comment by adrianhopebailie** [comment]
Fixed in #85
